Scripts/Python/Personal.py

```diff
@@ -111,6 +111,10 @@ def OnFirstUpdate(self):
         # this does not downgrade us if we have a normal KI :)
         PtSendKIMessageInt(kUpgradeKILevel, kMicroKI)
 
+        # Clear any private chat that failed to clear properly
+        PtClearPrivateChatList(PtGetLocalAvatar().getKey())
+        PtSendKIMessageInt(kUnsetPrivateChatChannel, 0)
+
```

Comment:
It might make more sense to put this in `xKI`'s `OnServerInitComplete()` so it will get cleared out for any links to new Ages (ie via the console).

Reply:
Will this break the case where people go into a neighbourhood egg room, close the door, and then link out? Right now I believe that's treated as an effective way to get long-running private chat with a select group of people

Reply:
Yeah, it does. I would consider the current behavior a bug, though - if you're not in the know about it, the behavior would probably be an unpleasant surprise (why can't anyone hear me?!?!?!).

Reply:
That is a bug, its supposed to clear itself when you link out.

Reply:
> It might make more sense to put this in `xKI`'s `OnServerInitComplete()` so it will get cleared out for any links to new Ages (ie via the console).

I though xKI only initializes once, but if it this runs every age link it would be a better place for it for sure.

Reply:
OnServerInitComplete runs every time new Age’s state information comes in :)

Reply:
Moved to xki/__init__.py instead.